taxonomies/additives_classes.txt

```diff
@@ -1320,10 +1391,37 @@ description:pt:Espessantes: substâncias que aumentam a viscosidade dos géneros
 description:sk:Zahusťovadlá sú látky, ktoré zvyšujú viskozitu potraviny.
 description:sl:Sredstva za zgostitev (gostila) so snovi, ki povečujejo viskoznost živila.
 description:sv:Förtjockningsmedel: ämnen som ökar ett livsmedels viskositet.
+description:bg:Загъстители са вещества, които увеличават вискозитета на хранителни продукти.
+description:ca:Espessidors són substàncies que augmenten la viscositat d'un aliment.
+description:ja:増粘剤は、食品の粘度を増加させる物質です。
+description:ru:Загустители - это вещества, которые увеличивают вязкость пищевого продукта.
+description:zh:增稠剂是增加食品粘度的物质。
 
 <en:thickener
 en:natural thickener
-pl:naturalne substancje zagęszczające, naturalna substancja zagęszczająca
+bg:естествен загъстител
+cs:přírodní zahušťovadlo
+de:natürliches Verdickungsmittel
+el:φυσικό πυκνωτικό μέσο
+es:espesante natural
+et:looduslik paksendaja
+fi:luonnollinen sakeutusaine
+fr:épaississant naturel
+hr:prirodno zgušnjivač
+hu:natúr sűrítőanyag
+it:addensante naturale
+ja:天然増粘剤
+lt:natūralus tirštiklis
+lv:dabīgs biezinātājs
+mt:ħasil naturali
+nl:natuurlijk verdikkingsmiddel
+pl:naturalny zagęszczacz
```

Comment:
The one you're removing is used (albeit rarely), the new one is not, but might be in the future.
Can you add it without removine old ones? :slightly_smiling_face:

Reply:
fixed